src/CarbonAware.Aggregators/test/CarbonAwareAggregatorTests.cs

```diff
@@ -344,6 +343,7 @@ public async Task<double> CalculateAverageCarbonIntensityAsync_ValidTimeInterval
 
     [TestCase("2021-11-18T00:00:00Z", null, TestName = "AverageCI valid startTime, null endTime")]
     [TestCase(null, "2021-11-18T00:00:00Z", TestName = "AverageCI null startTime, valid endTime")]
+    [TestCase("2021-11-19T00:00:00Z", "2021-11-18T00:00:00Z", TestName = "AverageCI startTime greater than endTime")]
```

Comment:
agreed, adding a test, and ensuring start==end is not allowed -- chatted with bderusha about it since prior API in other controller allowed start==end, but we agree this one shouldn't